src/RefractAPI.cc

```diff
@@ -73,7 +79,48 @@ namespace
 
         return std::move(element);
     }
-}
+
+    bool isRequest(const NodeInfo<snowcrash::Action>& action)
+    {
+        return !action.isNull() && !action.node->method.empty();
+    }
+
+    NodeInfoByValue<snowcrash::Asset> renderPayloadBody(
```

Comment:
Isn't it better to move these two functions to Render.cpp?

Reply:
We don't need these functions anywhere else, why would we have it declared in a header (`Render.h`)?